src/helpers/deepLinking.ts

```diff
@@ -20,41 +24,50 @@ const urlSafeBase64Decode = (data: string) => {
 };
 
 export const getInviteLink = (contact: InvitedContact, profileName: string): string => {
+    return `${BASE_URL}${INVITE}${makeInviteParams(contact, profileName)}`;
+};
+
+export const getInviteLinkWithParams = (params: string): string => {
+    return `${BASE_URL}${INVITE}${params}`;
+};
+
+const makeInviteParams = (contact: InvitedContact, profileName: string): string => {
     const randomSeedBytes = hexToUint8Array(contact.randomSeed);
-    const contactPulicKeyBytes = hexToUint8Array(contact.contactIdentity.publicKey);
+    const contactPublicKeyBytes = hexToUint8Array(contact.contactIdentity.publicKey);
     const base64RandomSeed = urlSafeBase64Encode(randomSeedBytes);
-    const base64ContactPublicKey = urlSafeBase64Encode(contactPulicKeyBytes);
-    const inviteLinkWithoutUsername = getInviteLinkWithBase64Params(base64RandomSeed, base64ContactPublicKey);
-    return `${inviteLinkWithoutUsername}/${encodeURIComponent(profileName)}`;
+    const base64ContactPublicKey = urlSafeBase64Encode(contactPublicKeyBytes);
+    return `\
+${INVITE_CODE_VERSION}${SEPARATOR}\
+${base64RandomSeed}${SEPARATOR}\
+${base64ContactPublicKey}${SEPARATOR}\
+${contact.createdAt + CONTACT_EXPIRY_THRESHOLD}${SEPARATOR}\
```

Comment:
Dates in Javascript are stored as milliseconds, however we don't really need that kind of precision. I think it would be better to store date in seconds and it would make the link a bit shorter too.
In that case the parsing code needs to be changed too.